adapters/wazirx-nft-Xauction-bsc/index.js

```diff
@@ -0,0 +1,176 @@
+require("dotenv").config();
+
+const web3 = require("web3");
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const TRANSFER_TOPIC = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const { AVAILABLE_PROTOCOLS } = require("../../sdk/constants");
+
+class WazirxNft {
+    constructor() {
+        this.name = "WazirXNFT";
+        this.symbol = "WRXNFT";
+        this.currencyCode = {
+            "0": {
+                symbol: "WRX",
+                decimals: 8,
+                token_address: "0x8e17ed70334c87ece574c9d537bc153d8609e2a3",
+            },
+            "1": {
+                symbol: "WRX",
+                decimals: 8,
+                token_address: "0x8e17ed70334c87ece574c9d537bc153d8609e2a3",
+            },
+            "2": {
+                symbol: "BNB",
+                decimals: 18,
+                token_address: "bnb",
+            },
+        };
+        this.token = "0x0000000000000000000000000000000000000000";
+        this.protocol = "bsc";
+        this.token_symbol = "bnb";
+        this.block = 14571030;
+        this.nftContract = "0x23Cad0003e3A2b27b12359B25c25dD9a890AF8e1";
+        this.contract = "0x12E2EA4ADb0C384FB273279c9E169E2f494CcA21";
```

Comment:
Please lowercase the value of `this.contract`.

Reply:
Ok